src/Bootstrap.php

```diff
@@ -0,0 +1,37 @@
+<?php
+declare(strict_types=1);
+
+namespace LotGD\Core;
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Mapping\AnsiQuoteStrategy;
+use Doctrine\ORM\Tools\Setup;
+use Doctrine\ORM\Tools\SchemaTool;
+
+class Bootstrap
+{
+    public static function game(): Game
```

Comment:
createGame or bootGame would convey this function better than just "game" which returns a GameObject. By reading only the function signature, one could think that this just returns the current game instance. Instead, it boots the entire game.